src/main/java/co/cask/hydrator/plugin/batchSource/KafkaBatchSource.java

```diff
@@ -304,7 +304,10 @@ public Schema getMessageSchema() {
     }
 
     public void validate() {
-      getBrokerMap();
```

Comment:
This is a required property and its value can not be empty or null, and we do not have a default value for this since we dont know which broker the user wants to use.